examples/supermassive-todomvc/package.json

```
@@ -3,9 +3,11 @@
   "main": "index.js",
   "version": "1.0.0",
   "scripts": {
+    "start": "concurrently 'yarn generate-graphql-interfaces --watch' 'yarn serve --progress --color --mode=development'",
     "build": "webpack --mode production --config webpack.config.ts",
-    "start": "webpack serve --progress --color --mode=development  --config webpack.config.ts",
-    "extract-schema": "supermassive extract-schema ./src/schema/typeDefs.graphql"
+    "serve": "webpack serve --config webpack.config.ts",
+    "extract-schema": "supermassive extract-schema ./src/schema/typeDefs.graphql",
+    "generate-graphql-interfaces": "relay-compiler --language graphitation --schema ./src/schema/typeDefs.graphql --src ./src"
```

```suggestion
    "relay": "relay-compiler --language graphitation --schema ./src/schema/typeDefs.graphql --src ./src"
```

Let's keep this simple for now. This is also consistent with what I've been calling these tasks in my branch, so when/if we want to rename them we can easily do so in a single sweep.